### PR TITLE
Remove extraneous !important, fix some scaling issues

### PR DESCRIPTION
**Pull request by danni**

### CODE REVIEW

File: src/components/StaticMap.js [hunk 1]
```diff
@@ -41,9 +35,18 @@ class Address extends React.Component {
         } ${
             this.props.location.streetAddressLine2()
         }`;
+
+        let size;
+
+        try {
+            size = window.innerWidth - 30;
```
Comment (DanielHeath) [inline src/components/StaticMap.js]: Don't we have a max-width on the container (800px or so IIRC)?

Comment (DanielHeath) [inline src/components/StaticMap.js]: Also the api won't provide bigger than 640 even if you ask for it.

**Comment by DanielHeath** [comment]
++ other than the image size thing

File: src/components/StaticMap.js (hunk 1 above)
Comment (danni) [inline src/components/StaticMap.js]: I'm thinking of screens smaller than 640.

**Pull request merged**